### ISSUE TITLE
Smart Table: Column Headers Should Be Keyboard Accessible

### ISSUE THREAD

**Issue by mcook103** [coreui/coreui-react#376]
For those who are visually impaired, it is crucial to be able to use a keyboard to tab through the table's header and data columns and toggle actions (sorting, etc.) using a key such as Enter or the Up/Down Arrows.

**Comment by mrholek** [comment]
@mcook103 it will be available in v5

**Comment by mcook103** [comment]
Great! Thank you.
